recipes/trec_eval/bld.bat

```diff
@@ -0,0 +1,8 @@
+@echo on
+
+make
+if %ERRORLEVEL% neq 0 exit 1
```

Comment:
Can we run the quicktest on Windows too?

Reply:
No, because windows cannot simply execute (all) bash scripts and doesn't have something like `diff`.

Of note, I'm running the equivalent of `quicktest` in the test section as well (also on windows, though in reduced form, again due to those script incompatibilities).

Reply:
While we could use some of the msys2 pkgs for that I'm ok leaving that for the future.